snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/event/ApplicationInstallTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.snowplowanalytics.snowplow.event
```

Comment:
Btw, let's make a task for adding missing copyright headers in the tracker files – they are missing quite randomly. Also we should update the year in copyrights.

Reply:
I've made an issue for it